packages/material-ui-codemod/codemod.js

```diff
@@ -5,7 +5,8 @@ const { promises: fs } = require('fs');
 const path = require('path');
 const yargs = require('yargs');
 
-const jscodeshiftExecutable = require.resolve('.bin/jscodeshift');
+const jscodeshiftDirectory = path.dirname(require.resolve('jscodeshift'));
+const jscodeshiftExecutable = path.join(jscodeshiftDirectory, 'bin/jscodeshift.js');
```

Comment:
Which would break if `jscodeshift` moves it. The previous code was taken from https://github.com/reactjs/react-codemod. Does https://github.com/reactjs/react-codemod also fail in windows?

Reply:
https://github.com/reactjs/react-codemod doesn't use Lerna.

Reply:
Why does it matter what they use?

Reply:
As I wrote, on Windows, Lerna does not create symlinks from node_modules/.bin (by design, it seems, as they use [cmd-shim](https://github.com/npm/cmd-shim)). As react-codemod doesn't use Lerna, it does not suffer from this issue.

> Which would break if jscodeshift moves their binary

I made it more future-proof.